### PR TITLE
Increase wait time for clone in the test test_pvc_to_pvc_clone

### LINKED ISSUE #9171: test_pvc_to_pvc_clone[CephFileSystem-None] failing while waiting for cloned PVC
The test case given below is failing while waiting for clones PVC to reach Bound state.
tests/manage/pv_services/pvc_clone/test_pvc_to_pvc_clone.py::TestClone::test_pvc_to_pvc_clone[CephFileSystem-None] (4.14 release branch)
logs - https://reportportal-ocs4.apps.ocp-c1.prod.psi.redhat.com/ui/#ocs/launches/557/17690/862542/862784/862786/log

### PR DESCRIPTION
**Pull request by jilju**
Fixes #9171 which is an intermittent issue.

### CODE REVIEW

**Comment by petr-balogh** [comment]
Hey @jilju , what is `cloen` in the commit message and titel of this PR?  From the change I see it's probably a typo and supposed to be `cloned`?

Please fix in commit and in title as well please as I think title will not change when you force push amended comment.

Thanks

**Comment by jilju** [comment]
> Hey @jilju , what is `cloen` in the commit message and titel of this PR? From the change I see it's probably a typo and supposed to be `cloned`?
> 
> Please fix in commit and in title as well please as I think title will not change when you force push amended comment.
> 
> Thanks

Thanks Petr for highlighting this typo. Fixed the commit message.

File: tests/functional/pv/pvc_clone/test_pvc_to_pvc_clone.py [hunk 1]
```diff
@@ -106,7 +106,9 @@ def test_pvc_to_pvc_clone(self, interface_type, setup, teardown_factory):
             sc_name, parent_pvc, clone_yaml, namespace
         )
         teardown_factory(cloned_pvc_obj)
-        helpers.wait_for_resource_state(cloned_pvc_obj, constants.STATUS_BOUND)
+        helpers.wait_for_resource_state(
+            cloned_pvc_obj, constants.STATUS_BOUND, timeout=120
```
Comment (vavuthu) [inline tests/functional/pv/pvc_clone/test_pvc_to_pvc_clone.py:110]: could you increase to 300, whenever there is load on cephfs it takes more time to bound

**Review by vavuthu — APPROVED** [review]

File: tests/functional/pv/pvc_clone/test_pvc_to_pvc_clone.py (hunk 1 above)
Comment (jilju) [reply tests/functional/pv/pvc_clone/test_pvc_to_pvc_clone.py]: Done

**Review by nehaberry — APPROVED** [review]

**Comment by petr-balogh** [comment]
> > Hey @jilju , what is `cloen` in the commit message and titel of this PR? From the change I see it's probably a typo and supposed to be `cloned`?
> 
> > 
> 
> > Please fix in commit and in title as well please as I think title will not change when you force push amended comment.
> 
> > 
> 
> > Thanks
> 
> 
> 
> Thanks Petr for highlighting this typo. Fixed the commit message.

Hey @jilju , I don't see it's fixed in original commit. I meant you to 'git commit --amend' and update original commit, instead I see 2 commits but first one still has typo in.

Now you will need to probably squash commits and fix commit message there.

The title of PR also still showing typo. This you can edit here in web UI of github somewhere next to title please.

**Review by petr-balogh — APPROVED** [review]

**Pull request merged**